aiven/client/cli.py

```diff
@@ -3059,6 +3059,93 @@ def mirrormaker__replication_flow__delete(self):
             self.args.target_cluster,
         )
 
+    @arg.project
+    @arg.service_name
+    def service__flink__table__list(self):
+        """List Flink tables"""
+        project_name = self.get_project()
+        self.print_response(self.client.list_flink_tables(project_name, self.args.service_name))
+
+    @arg.project
+    @arg.service_name
+    @arg("integration_id", help="Service integration ID")
+    @arg("-n", "--table-name", required=True, help="Table name")
+    @arg("--kafka-topic", required=False, help="Topic name, used as a source/sink. (Kafka integration only)")
+    @arg("--jdbc-table", required=False, help="Table name in Database, used as a source/sink. (PG integration only)")
+    @arg("-p", "--partitioned-by", required=False, help="A column from a schema, table will be partitioned by")
+    @arg(
+        "-l",
+        "--like-options",
+        required=False,
+        help="Clause can be used to create a table based on a definition of an existing table"
+    )
+    @arg("-s", "--schema-sql", required=True, help="Source/Sink table schema")
+    def service__flink__table__create(self):
+        """Create a Flink table"""
+        project_name = self.get_project()
+        self.print_response(
+            self.client.create_flink_table(
+                project_name,
+                self.args.service_name,
+                self.args.integration_id,
+                self.args.table_name,
+                self.args.schema_sql,
+                self.args.kafka_topic,
+                self.args.jdbc_table,
+                self.args.partitioned_by,
+                self.args.like_options,
+            )
+        )
+
+    @arg.project
+    @arg.service_name
+    @arg("table_id", help="Table ID")
+    def service__flink__table__get(self):
+        """Get a Flink table"""
+        project_name = self.get_project()
+        self.print_response(self.client.get_flink_table(
+            project_name,
+            self.args.service_name,
+            self.args.table_id,
+        ))
+
+    @arg.project
+    @arg.service_name
+    @arg("table_id", help="Table ID")
+    def service__flink__table__delete(self):
+        """Delete a Flink table"""
+        project_name = self.get_project()
+        self.print_response(self.client.delete_flink_table(
+            project_name,
+            self.args.service_name,
+            self.args.table_id,
+        ))
```

Comment:
It would be nice to print something like `deleted` (on its own line like `service topic-delete`). 
 It seems this might also need some API changes as API for Kafka topics returns json object with `{"message": "deleted"}`

Reply:
Fair

Reply:
I add some temporary fix, that do not need a changes from API